charts/dapr/README.md

```diff
@@ -174,6 +174,7 @@ The Helm chart has the follow configuration options that can be supplied:
 ### Dapr Sidecar Injector options:
 | Parameter                                 | Description                                                             | Default                 |
 |-------------------------------------------|-------------------------------------------------------------------------|-------------------------|
+| `dapr_sidecar_injector.enable`            | Enable the sidecar injector                                             | `enable`                |
```

Comment:
nitpick... 
`dapr_sidecar_injector.enabled`

Reply:
Thanks for the catch!